actors/builtin/reward/reward_state.go

```diff
@@ -14,14 +14,30 @@ type NetworkTime = big.Int
 type Spacetime = big.Int
 
 type State struct {
-	BaselinePower        abi.StoragePower
-	RealizedPower        abi.StoragePower
-	CumsumBaseline       Spacetime
-	CumsumRealized       Spacetime
+	// Baseline network power in raw bytes (fully deterministic & functionally equivalent to a hard-coded
+	// lookup table), above which, full reward minting will be possible.
```

Comment:
This and the one below are "most recent value, calculated at end-of-epoch cron", which is useful context.